src/components/Calendar.css

```diff
@@ -0,0 +1,156 @@
+.react-calendar {
```

Comment:
since this .css file is not a component, we shouldn't write it in capital letter. If this is supposed to be attached to Calendar.jsx component, then I recommend placing it in a new folder /components/Calendar alongside the component itself.